charts/langsmith/docs/DELETE-TRACES.md

```diff
@@ -0,0 +1,61 @@
+# Deleting Traces
+
+The LangSmith UI does not currently support the deletion of an invidual trace. This, however, can be accomplished by directly removing the trace from all materialized views in ClickHouse (except the runs_history views) and the runs and feedback tables themselves.
```

Comment:
worth adding a note that deleting traces does not affect billing? what about usage metrics?

Reply:
All this is for self hosted, so none of the billing constructs necessarily apply, at least not in today's world.